test/unit/tile.js

```diff
@@ -192,6 +217,26 @@ describe('Tile', function () {
     });
   });
 
+  it('Deep Zoom layout with custom size+angle', function (done) {
+    const directory = fixtures.path('output.512_90.dzi_files');
+    rimraf(directory, function () {
+      sharp(fixtures.inputJpg)
+        .tile({
+          size: 512,
+          angle: 90
+        })
+        .toFile(fixtures.path('output.512_90.dzi'), function (err, info) {
+          if (err) throw err;
+          assert.strictEqual('dz', info.format);
+          assert.strictEqual(2725, info.width);
+          assert.strictEqual(2225, info.height);
+          assert.strictEqual(3, info.channels);
+          assert.strictEqual('undefined', typeof info.size);
+          assertDeepZoomTiles(directory, 512, 13, done);
+        });
```

Comment:
I wonder if there's a reliable way to test that the tiles have actually been rotated by 90 degrees? Perhaps inspect one tile against a known fixture?

Reply:
this may work, what do you think?